src/ray/raylet/worker_pool.h

```diff
@@ -397,6 +392,15 @@ class WorkerPool : public WorkerPoolInterface {
 
   /// This map tracks the latest infos of unfinished jobs.
   absl::flat_hash_map<JobID, rpc::JobConfig> unfinished_jobs_;
+
+  /// The pool of idle non-actor workers of all languages. This is used to kill idle
+  /// workers in FIFO order.
+  std::list<std::shared_ptr<WorkerInterface>> idle_of_all_languages;
```

Comment:
Could we add the size of these two lists to DebugString() for debug purposes?

Reply:
Done.